### PR TITLE
Initial Peribolos configuration for knative and knative-sandbox

### PR DESCRIPTION
**Pull request by evankanderson**
Fixes https://github.com/knative/test-infra/issues/148

This is a cleanup of [`peribolos --dump knative`](https://github.com/kubernetes/test-infra/tree/master/prow/cmd/peribolos) with cleanup and addition of the org header.

Note that I *did not* merge the two files because I'm guessing that we want to start applying the `knative-sandbox` configuration sooner (probably I'll do it in the next day or two), and apply the `knative` configuration once we have confidence in `knative-sandbox`.

/assign @mattmoor @bsnchan @isdal @pmorie @chizhg

### CODE REVIEW

File: peribolos/knative-sandbox.yaml [hunk 1]
```diff
@@ -0,0 +1,655 @@
...
+            description: The Working Group leads for Autoscaling
+            members:
+            - vagababov
+            - markusthoemmes
+          Networking WG Leads:
+            description: The Working Group leads for Networking
+            members:
+            - tcnghia
+            - mdemirhan
+
```
Comment (mattmoor) [inline peribolos/knative-sandbox.yaml:655]: Remove trailing whitespace:
```suggestion

```

File: peribolos/knative.yaml [hunk 2]
```diff
@@ -0,0 +1,1276 @@
...
+            description: The Working Group leads for Autoscaling
+            members:
+            - vagababov
+            - markusthoemmes
+          Networking WG Leads:
+            description: The Working Group leads for Networking
+            members:
+            - tcnghia
+            - mdemirhan
+
```
Comment (mattmoor) [inline peribolos/knative.yaml:1276]: Remove trailing whitespace:
```suggestion

```

File: peribolos/knative-sandbox.yaml [hunk 3]
```diff
@@ -0,0 +1,656 @@
...
```
Comment (mattmoor) [inline peribolos/knative-sandbox.yaml:656]: Remove trailing whitespace:
```suggestion

```

File: peribolos/knative.yaml (hunk 2 above)
Comment (mattmoor) [inline peribolos/knative.yaml:1276]: Remove trailing whitespace:
```suggestion

```

**Comment by dprotaso** [comment]
If you made them one file you could use yaml anchors to de-dupe the list

**Comment by evankanderson** [comment]
> If you made them one file you could use yaml anchors to de-dupe the list

I thought about this, but I'm not sure that I want to use yaml anchors. I am thinking of possibly making sub-directories (and possibly including OWNERS delegation, then) and assembling the components via includes.

**Comment by evankanderson** [comment]
> > If you made them one file you could use yaml anchors to de-dupe the list
> 
> I thought about this, but I'm not sure that I want to use yaml anchors. I am thinking of possibly making sub-directories (and possibly including OWNERS delegation, then) and assembling the components via includes.

One advantage of the "compose from small files" approach is that it's possible to diff the resulting generated file as part of the PR and make sure that the result is still sane.

File: peribolos/knative-sandbox.yaml [hunk 4]
```diff
@@ -0,0 +1,656 @@
...
+            - mattmoor
+            - evankanderson
+            - vaikas
+      Knative Milestone Maintainers:
+        description: 'DO NOT RENAME - this group is used by Prow to control who can manipulate
+          milestones: https://github.com/knative/test-infra/blob/64615f92c4ebdd4e4423015ffc0db45877660a0d/ci/prow/plugins.yaml#L49'
+        maintainers:
+        - mattmoor
+        - evankanderson
+        - vaikas
```
Comment (mattmoor) [inline peribolos/knative-sandbox.yaml:588]: I wonder, can we use labels and anchors now that this is yaml?

File: peribolos/knative.yaml [hunk 5]
```diff
@@ -0,0 +1,1276 @@
...
+              issues and milestones as we sort out roles
+            members:
+            - akashrv
+      Knative Admin:
+        description: Individuals in the project who are responsible for billing and general
+          project administration.
+        maintainers:
+        - isdal
+        - pmorie
+        - rgregg
```
Comment (mattmoor) [inline peribolos/knative.yaml:708]: 😢 remove?

**Comment by mattmoor** [comment]
Two other thoughts:
1. Is there a dry-run for peribolos that we can use to produce a diff?  If there is, then I think having a bot that posts that diff to alterations here would be extremely useful.

2. Add OWNERS for the new directory, so TOC can help steering manage this (as with the raw github ACLs)?

File: peribolos/knative-sandbox.yaml [hunk 6]
```diff
@@ -0,0 +1,662 @@
...
+            description: The Working Group leads for Autoscaling
+            members:
+            - vagababov
+            - markusthoemmes
+          Networking WG Leads:
+            description: The Working Group leads for Networking
+            members:
+            - tcnghia
+            - mdemirhan
+
```
Comment (mattmoor) [inline peribolos/knative-sandbox.yaml:662]: Remove trailing whitespace:
```suggestion

```

File: peribolos/knative.yaml [hunk 7]
```diff
@@ -0,0 +1,1285 @@
...
+            description: The Working Group leads for Autoscaling
+            members:
+            - vagababov
+            - markusthoemmes
+          Networking WG Leads:
+            description: The Working Group leads for Networking
+            members:
+            - tcnghia
+            - mdemirhan
+
```
Comment (mattmoor) [inline peribolos/knative.yaml:1285]: Remove trailing whitespace:
```suggestion

```

File: peribolos/knative-sandbox.yaml [hunk 8]
```diff
@@ -0,0 +1,651 @@
...
```
Comment (mattmoor) [inline peribolos/knative-sandbox.yaml:651]: Remove trailing whitespace:
```suggestion

```

File: peribolos/knative.yaml (hunk 7 above)
Comment (mattmoor) [inline peribolos/knative.yaml:1285]: Remove trailing whitespace:
```suggestion

```

File: peribolos/knative-sandbox.yaml (hunk 8 above)
Comment (mattmoor) [inline peribolos/knative-sandbox.yaml:651]: Remove trailing whitespace:
```suggestion

```

File: peribolos/knative.yaml (hunk 7 above)
Comment (mattmoor) [inline peribolos/knative.yaml:1285]: Remove trailing whitespace:
```suggestion

```

**Comment by evankanderson** [comment]
I've also removed all the extra members from knative-sandbox that haven't accepted the invite

> Two other thoughts:
> 
> 1. Is there a dry-run for peribolos that we can use to produce a diff?  If there is, then I think having a bot that posts that diff to alterations here would be extremely useful.

There is a dry-run for Peribolos, but it's noisy. Example here:
https://gist.githubusercontent.com/evankanderson/59115fb8ffc02d2323f41b8d912e4779/raw/20f4fc3455bf978e4e37762a7f78818ed9492849/peribolos-output.json

Maybe it's worth cleaning this up in test-infra or figuring out the right way to post-process with `jq`.

> 2. Add OWNERS for the new directory, so TOC can help steering manage this (as with the raw github ACLs)?

Good point! Done.

File: peribolos/knative-sandbox.yaml [hunk 9]
```diff
@@ -0,0 +1,355 @@
...
```
Comment (mattmoor) [inline peribolos/knative-sandbox.yaml:355]: Remove trailing whitespace:
```suggestion

```

File: peribolos/knative.yaml (hunk 7 above)
Comment (mattmoor) [inline peribolos/knative.yaml:1285]: Remove trailing whitespace:
```suggestion

```

**Comment by chizhg** [comment]
/lgtm
Please note I have staged a PR for adding peribolos Prow jobs in test-infra repo - https://github.com/knative/test-infra/pull/2060

**Comment by evankanderson** [comment]
https://github.com/kubernetes/test-infra/pull/17895 makes the diff substantially smaller and easier to read when comparing `--dump-full --dump` output to the input file.

**Pull request merged**